docs/csharp/language-reference/tokens/raw-string.md

```diff
@@ -0,0 +1,37 @@
+---
+description: "Raw string literals can contain any arbitrary text without the need for special escape sequences.You begin and end a raw string literal with a minimum of three double-quote characters."
+title: "Raw string literals = \"\"\""
+ms.date: 12/08/2022
```

Comment:
@Youssef1313 @CyrusNajmabadi 

Is there a text token for F1 help for the `"""` token in Visual Studio? If so, I'll add it here.

Reply:
@BillWagner Yes, it's `RawStringLiteral_CSharpKeyword`.

It was added to https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/builtin-types/reference-types.md, so it should be removed from there and kept only in `raw-string.md`